lib/reindex/s3_bucketd.py

```diff
@@ -443,11 +444,19 @@ def log_report(resource, name, obj_count, total_size):
                 update_report(account_reports, total.bucket.userid, total.obj_count, total.total_size)
 
             # Bucket reports can be updated as we get them
-            pipeline = redis_client.pipeline(transaction=False)  # No transaction to reduce redis load
-            for bucket, report in bucket_reports.items():
-                update_redis(pipeline, 'buckets', bucket, report['obj_count'], report['total_size'])
-                log_report('buckets', bucket, report['obj_count'], report['total_size'])
-            pipeline.execute()
+            if options.dry_run:
+                for bucket, report in bucket_reports.items():
+                    _log.warning(
+                        "DryRun: resource buckets [%s] will be not updated with obj_count %i and total_size %i" % (
+                            bucket, report['obj_count'], report['total_size']
```

Comment:
I would suggest `_log.info`, as it is a dry-run it is expected by the user.

Also thinking the formulation is somewhat odd or sounds negative and could be improved but I don't have a good substitution in mind at the moment. Thinking that the message should just convey what are the values that would be updated could work better (maybe just rewording as `would be updated` could work).

Reply:
Just another proposition:

`DryRun: obj_count %i and total_size %i was calculated for resource bucket [%s]. The bucket has not been updated.`